bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/cache/ExpiringCacheMap.java

```diff
@@ -35,6 +36,7 @@
  * @param <K> the type of the key
  * @param <V> the type of the value
  */
+@NonNullByDefault
```

Comment:
Then please remove the `null` checks from the constructor and also remove the tests with the null values

Reply:
Done.